docs/fundamentals/integrations-overview.md

```diff
@@ -56,6 +56,12 @@ When you add a client integration to a project within your .NET Aspire solution,
 - **[Health checks](health-checks.md)**: Exposes HTTP endpoints to provide basic availability and state information about an app. Health checks are used to influence decisions made by container orchestrators, load balancers, API gateways, and other management services.
 - **[Resiliency](/dotnet/core/resilience/http-resilience)**: The ability of your system to react to failure and still remain functional. Resiliency extends beyond preventing failures to include recovering and reconstructing your cloud-native environment back to a healthy state.
 
+## Versioning considerations
+
+Hosting and client integrations are updated each release to target the latest stable versions of dependent resources. When container images are updated with new image versions, the hosting integrations update to these new versions. Similarly, when a new NuGet version is available for a dependent client library, the corresponding client integration updates to the new version. This ensures the latest features and security updates are available to applications.
```

Comment:
I think we should make "each release" more defined. It's not clear to me if that means a servicing release of a hosting integration that brings in a container will update to a new major version of the image tag, or if that only happens in minor (or major) releases of the hosting integration package. Same goes for the client integration packages.